File: Session.cpp
#include "Session.hpp"


void Session::startSession() {

    actualDuration = 0;
}


void Session::logExercise(const ExerciseLog& exercise) {
    completedExercises.push_back(exercise);
}


void Session::completeSession() {
    burnedCalories = completedExercises.size() * 100;
}


int Session::getBurnedCalories() const {
    return burnedCalories;
}

File: FitnessAppCsharp/Session.cs
using System;
using System.Collections.Generic;

namespace FitnessApp
{
    public class Session
    {
        private string id;
        private string programId;
        private string scheduledDate;
        private int actualDuration;
        private int burnedCalories;
        private List<ExerciseLog> completedExercises = new List<ExerciseLog>();

        public void StartSession()
        {
            actualDuration = 0;
        }

        public void LogExercise(ExerciseLog exercise)
        {
            completedExercises.Add(exercise);
        }

        public void CompleteSession()
        {
            burnedCalories = completedExercises.Count * 100;
        }

        public int GetBurnedCalories()
        {
            return burnedCalories;
        }
    }
}